src/config.rs

```diff
@@ -26,17 +31,34 @@ impl DatabaseLocation {
     }
 }
 
-pub fn deserialize_self_delay<'de, D: Deserializer<'de>>(deserializer: D) -> Result<u64, D::Error> {
-    let num = u64::deserialize(deserializer)?;
+pub fn deserialize_self_delay<'de, D: Deserializer<'de>>(
+    _deserializer: D,
+) -> Result<u64, D::Error> {
+    #[cfg(feature = "allow_custom_self_delay")]
+    {
+        let num = u64::deserialize(_deserializer)?;
+        if num < 10 {
+            return Err(de::Error::invalid_value(
+                de::Unexpected::Unsigned(num as u64),
+                &"at least 10",
+            ));
+        }
 
-    if num < 10 {
-        return Err(de::Error::invalid_value(
-            de::Unexpected::Unsigned(num as u64),
-            &"at least 10",
-        ));
+        if num < 120 {
```

Comment:
This is always going to be testing, right? Shouldn't we remove this warning completely?

Reply:
Good call, removed.